src/screens/Home/components/MaintenanceMesg/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { CTFragment, CTText } from 'layout';
+
+function MaintenanceMesg() {
+  return (
+    <CTFragment styles={{ background: 'teal' }} justConCenter padding="15">
+      <CTText white bold size="medium">
+        ClassTranscribe will be unavailable for a few hours 
```

Comment:
do you want to change this in a additional commit to be "" or turned off.

Reply:
Sure, I will make the change in another PR